.github/CONTRIBUTING_GUIDELINES.md

```diff
@@ -69,6 +88,29 @@
 
   at this address the web application will be running
 
+### Step 4 (Optional)
+
+If you want to use MongoDB, you can either
```

Comment:
If you want to use MongoDB (instead of the default JSON storage), you can either:
- Download MongoDB locally and run it like a background process (Ideal for Windows and Mac users)
  - ......
- Use Docker to run MongoDB so that it remains isolated from the rest of the system. (Ideal for Linux users)
   - .....